src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -1592,7 +1596,7 @@ export class SearchView extends ViewPane {
 
 			this.messageDisposables.push(dom.addDisposableListener(openInEditorLink, dom.EventType.CLICK, (e: MouseEvent) => {
 				dom.EventHelper.stop(e, false);
-				this.instantiationService.invokeFunction(createEditorFromSearchResult, this.searchResult, this.searchIncludePattern.getValue(), this.searchExcludePattern.getValue());
+				this.instantiationService.invokeFunction(createEditorFromSearchResult, this.searchResult, this.searchIncludePattern?.getValue(), this.searchExcludePattern?.getValue()); // {{SQL CARBON EDIT}}
```

Comment:
Why did you need to change this? Shouldn't you be creating the inputSearchPattern in your implementation?

Reply:
because we don't have the searchIncludePattern and searchExcludePattern on our searchView. Those are the options  on search view which lets you specify files to include and files to exclude.

Reply:
Is there a reason we wouldn't want those at some point? They seem like useful features so what if instead our search view just created empty ones for now that we could hook up later if we wanted?

Reply:
Especially since you have the config values defined above for exclude patterns

Reply:
yes, we'll have an option to have these but they'll not be part of notebook search view. SearchView in vscode is one integrated view with everything in one ViewPane. But in our case, search widget is added to the view pane container and the results, notebook contents are separate viewlets that are added to the view pane container and not aware of the search widget. These include/exclude pattern widgets would be part of the search widget not search view.